src/main/java/org/spongepowered/api/resource/pack/PackInfo.java

```diff
@@ -86,4 +80,22 @@
      */
     boolean isRemote();
 
+    boolean isLocked();
+
+    Priority getPriority();
+
+    Compatibility getCompatability();
+
+    enum Priority {
+        FIRST,
+        LAST
+    }
+
+    enum Compatibility {
+        OLD,
+        NEW,
+        OK
```

Comment:
Consider using some other thing like a string version, like what we do for `GameVersion`